backend/src/main/java/com/woowacourse/kkogkkog/application/MemberService.java

```diff
@@ -28,4 +33,12 @@ public Long save(MemberCreateRequest memberCreateRequest) {
 
         return memberRepository.save(member).getId();
     }
+
+    public MembersResponse findAll() {
+        List<MemberResponse> memberResponses = memberRepository.findAll().stream()
+            .map(it -> new MemberResponse(it.getId(), it.getEmail(), it.getNickname()))
+            .collect(toList());
```

Comment:
email과 nickname 모두 String 타입이라 순서 실수하기 좋아보여요. static 팩토리 메서드를 두는건 어떨까요

```suggestion
            .map(MemberResponse::of)
            .collect(toList());
```

Reply:
이거 놓친 부분이네요 👍